molecule/default/molecule.yml

```diff
@@ -10,6 +10,12 @@ platforms:
     image: debian:stretch-slim
   - name: debian10
     image: debian:buster-slim
+  - name: fedora29
+    image: fedora:29
+  - name: fedora30
+    image: fedora:30
+  - name: amazon2018.03
+    image: amazonlinux:2018.03
```

Comment:
Did you test this role on all of these operating systems?  If not, feel free to remove them.

Reply:
Pared it down to just Debian. Thanks for the feedback. 👍